src/corefx/System.Globalization.Native/calendarData.cpp

```diff
@@ -7,6 +7,7 @@
 #include <string.h>
 
 #include "locale.hpp"
+#include "calendarData.hpp"
```

Comment:
I noticed that in github.com/dotnet/corefx/src/Native the header files all have extension ".h".  Do we want to use .h or .hpp?